packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/productForm.js

```diff
@@ -71,14 +87,33 @@ const ProductForm = props => {
 
     return (
         <Fragment>
-            <FormError
+            <Dialog
                 classes={{
-                    root: classes.errorContainer
+                    body: classes.body,
```

Comment:
Since we removed the `body` class in from `productForm.css`, this override is going to pass in `undefined` and wipe out the `Dialog` 's default `body` styles. 

In this particular case, that causes the user to be unable to scroll within the dialog so they won't be able to click the "Submit" button 😬 

Please remove this line too 👍

Reply:
Fixed!